packages/atst/src/constants/defaultDataType.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * @internal
+ * Default data type for attestations
+ */
+export const DEFAULT_DATA_TYPE = 'string' as const
```

Comment:
shouldn't for this make more sense to be declared within `types/DataTypeOption` as it seams is the default of that?

Reply:
Yea I think you are right I will leave this comment open and do this before landing